centreon/tests/php/Core/Resources/Application/UseCase/FindHostsStatusCount/FindHostsStatusCountTest.php

```diff
@@ -85,7 +85,8 @@
         new UpStatusCount(2),
         new PendingStatusCount(4)
     );
-    $resourceStatusCount = new ResourcesStatusCount($hostStatusCount);
+    $servicesStatusCount = null;
+    $resourceStatusCount = new ResourcesStatusCount($hostStatusCount, $servicesStatusCount);
```

Comment:
Just to avoid possibly confusing variable assignation:
```suggestion
    $resourceStatusCount = new ResourcesStatusCount(
        hostsStatusCount: $hostStatusCount, 
        servicesStatusCount: null
    );
```